### DISCUSSION TITLE
KTSFDT1

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#390] [Show and tell]
### Channel ID

KTSFDT1.us

### Stream URL

https://cdn.jwplayer.com/manifests/DYuIIdNE.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://cdn.jwplayer.com/manifests/DYuIIdNE.m3u8
<img width="1412" alt="image" src="https://user-images.githubusercontent.com/7253922/232255960-79410346-e144-4e3a-afed-bca69c60ca2a.png">

**Comment by freearhey** [comment]
Looped video:
<img width="1415" alt="image" src="https://user-images.githubusercontent.com/7253922/232856923-fa473a3f-9eca-4b5d-9ddd-7099ab643822.png">
